MarsFrameworkSpecflow/MarsFrameworkSpecflow/Pages/ManageRequestsPage.cs

```diff
@@ -0,0 +1,434 @@
+﻿using OpenQA.Selenium;
+using static MarsFrameworkSpecflow.Global.GlobalDefinitions;
+using MarsFrameworkSpecflow.Global;
+
+namespace MarsFrameworkSpecflow.Pages
+{
+    public class ManageRequestsPage : Base
+    {
+        IWebElement manageRequestsDropdown => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[1]/div/div[1]"));
+        IWebElement receivedRequestPage => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[1]/div/div[1]/div/a[1]"));
+        IWebElement sentRequestPage => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[1]/div/div[1]/div/a[2]"));
+        IWebElement showRequestDetails => driver.FindElement(By.XPath("//*[@id=\"received-request-section\"]/div[2]/div[1]/table/tbody/tr[last()]/td[1]"));
+        IWebElement starRating => driver.FindElement(By.XPath("//*[@id=\"received-request-section\"]/div[2]/div[1]/table/tbody/tr[last()]/td[3]/div/i[5]"));
+        IWebElement acceptRequest => driver.FindElement(By.XPath("//*[@id=\"received-request-section\"]/div[2]/div[1]/table/tbody/tr[1]/td[8]/button[1]"));
+        IWebElement declineRequest => driver.FindElement(By.XPath("//*[@id=\"received-request-section\"]/div[2]/div[1]/table/tbody/tr[1]/td[8]/button[2]"));
+        IWebElement withdrawRequest => driver.FindElement(By.XPath("//*[@id=\"sent-request-section\"]/div[2]/div[1]/table/tbody/tr[1]/td[8]/button"));
```

Comment:
I would you to break this down into smaller components, Since your on the advance task I would you make use of the POM pattern properly